src/Client.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Http\Client;
+
+/**
+ * Interface for both HTTP Client type
+ *
+ * @author Márk Sági-Kazár mark.sagikazar@gmail.com>
+ */
+interface Client extends PsrHttpClient, HttpClient
```

Comment:
> the documentation should state clearly that you should specify PsrHttpClient or HttpClient rather than the combined interface unless you really need both.

Agreed